client/state/utils/test/actions.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+/**
+ * Internal dependencies
+ */
+import { UTILS_UNIQUEID_INCREMENT } from 'state/action-types';
+import { uniqueId, uniqueIdIncrement } from '../actions';
+
+describe( 'actions', () => {
+	describe( 'uniqueIdIncrement()', () => {
+		it( 'should return an action object with type UTILS_UNIQUEID_INCREMENT', () => {
+			const action = uniqueIdIncrement();
+			expect( action ).to.eql( {
+				type: UTILS_UNIQUEID_INCREMENT
+			} );
+		} );
+	} );
+
+	describe( 'uniqueId()', () => {
+		const getState = () => {
+			return {
+				utils: {
+					uniqueId: 3
+				}
+			}
+		};
+
+		it( 'should call dispatch to increment the counter', () => {
+			const dispatchSpy = spy(),
+				getStateSpy = spy( getState );
+
+			uniqueId()( dispatchSpy, getStateSpy );
+			expect( dispatchSpy.calledWithMatch( { type: UTILS_UNIQUEID_INCREMENT } ) ).to.be.ok;
+		} );
+
+		it( 'should return new counter value', () => {
+			const dispatchSpy = spy(),
+				getStateSpy = spy( getState ),
+				newValue = uniqueId()( dispatchSpy, getStateSpy );
+
+			expect( newValue ).to.eql( '3' );
```

Comment:
@dmsnell 
That would require importing both `actions.js` and a `reducer.js`. I dont want to do that in this file, because that would violate some benefits of unit tests (for example it would be hard to pinpoint what exactly has been broken, because tests would fail if either action file or reducer had an error)

But you're right, we should test that and I added separate 'integration' test in commit 2520c5d6d18698547ace4e9a096353c259594dd3